scripts/tags/video.js

```diff
@@ -0,0 +1,140 @@
+(function() {
+  'use strict';
+  
+  /**
+   * search index in an array with a regex
+   * @param {Array} array
+   * @param {Regex} regex
+   * @param {Number} startpos
+   * @return {Number}
+   */
+
+  var rPath = new RegExp(
+    '((([A-Za-z]{3,9}:(?:\\/\\/)?)(?:[-;:&=+$,\\w]+@)?' +
+    '[A-Za-z0-9.-]+|(?:www.|[-;:&=+$,\\w]+@)[A-Za-z0-9.-]+)' +
+    '((?:\\/[\\+~%\\/.\\w-_]*)?\\??(?:[-+=&;%@.\\w_]*)' +
+    '#?(?:[.!\\/\\w]*))|^[A-Za-z0-9_\\/-]+\\.\\w{2,4})');
+  var rClass = /^[_a-zA-Z0-9-]+$/;
+  var rSize = /^\d+(?:|\.\d+)(?:px|%)?$/;
+  var captionClass = 'caption';
+  var noCaptionClass = 'nocaption';
+  var clearClass = 'clear';
+  var autoplayClass = 'autoplay';
+  var loopClass = 'loop';
+  var mutedClass = 'muted';
+  var noControlsClass = 'nocontrols';
+  /**
+   * Video tag
+   *
+   * Syntax:
+   *     {% video [classes] /path/to/video
+   *     [width%] [title text] %}
```

Comment:
I suggest we name parameters without including their units so it's consistent with the other tags.

Reply:
Corrected, realized height was left out so that's been included